Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM node:lts-alpine
 
 RUN set -ex && mkdir /app
+RUN apk add --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN wget https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp -O /usr/local/bin/yt-dlp && chmod a+rx /usr/local/bin/yt-dlp
+RUN wget https://yt-dl.org/downloads/latest/youtube-dl -O /usr/local/bin/youtube-dl && chmod a+rx /usr/local/bin/youtube-dl
```

Comment:
There is a package of it for Alpine Linux, which we used it as our base image: https://pkgs.alpinelinux.org/package/edge/community/x86_64/yt-dlp

Install it with `apk install --no-cache yt-dlp`.

Reply:
There would be an error
```
/app # apk add --no-cache yt-dlp
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/aarch64/APKINDEX.tar.gz
ERROR: unable to select packages:
  yt-dlp (no such package):
    required by: world[yt-dlp]
```

Reply:
So I would keep the original method to install yt-dlp.